pages/blog/release-notes-2.2.0.md

```diff
@@ -0,0 +1,64 @@
+
+Now you can add a new **servers** property to [Channel Item Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#channel-item-object). It must be a list of server names provided as a string.
+
+```yaml
+description: This application publishes WebUICommand messages to an AMQP queue on RabbitMQ brokers in the Staging and Production environments.
+servers:
+  - rabbitmqBrokerInProd
+  - rabbitmqBrokerInStaging
+subscribe:
+  message:
+    $ref: "#/components/messages/WebUICommand"
+bindings:
+  amqp:
+    is: queue
+```
+
+Names of servers must match the names of the servers defined in the [Servers Object](https://github.com/asyncapi/spec/blob/2021-09-release/spec/asyncapi.md#serversObject). This new property is optional, so moving from 2.1.0 to 2.2.0 is as easy as changing the specification version in your current AsyncAPI file. Suppose **servers** is absent or empty. In that case, the given channel must be available on all servers defined in the Servers Object, as it was before the 2.2.0 version.
+
+For more details, check out [this pull request](https://github.com/asyncapi/spec/pull/531).
+
+We heard some community members asking for this feature. It was [Gerald Loeffler](https://www.linkedin.com/in/geraldloeffler/) that decided to champion the proposal and lead it until it got released. Thank you :pray:.
+
+## New protocol bindings
+
+The specification is now extended to support the following custom protocols through the bindings feature:
+- **Anypoint MQ**, thanks to [Gerald Loeffler](https://www.linkedin.com/in/geraldloeffler/). For more details check out [this pull request](https://github.com/asyncapi/spec/pull/545) and [binding definition](https://github.com/asyncapi/bindings/tree/master/anypointmq).
+
+## Become a contributor
+
+Pushing things through into the specification is not an easy process are requires a lot of time and patience, but it is worth it. Have a look at our [contribution guide](https://github.com/asyncapi/spec/blob/master/CONTRIBUTING.md) and start contributing.
+
+## Conclusions
+
+Are you wondering how we managed to release 2.2.0 just three months after 2.1.0? I recommend you familiarize yourself with the [AsyncAPI release process](https://github.com/asyncapi/spec/blob/master/RELEASE_PROCESS.md). The next release is scheduled for January 2022. Later releases are in April, June and September, according to the agreed [release cadence](https://github.com/asyncapi/spec/blob/master/RELEASE_PROCESS.md#release-cadence).
+
+<img className="w-3/4" src="/img/posts/release-notes-2.2.0/brace.webp" alt="Meme showing a knight, Ned Stark from Game of Thrones. Description says: Brace yourself, all the stars in heaven say 3.0.0 version is coming." />
+
+The above meme gives you mixed feelings? Are you afraid of possible changes or actually happy to see it coming? Don't overthink it, just join our [Slack](https://www.asyncapi.com/slack-invite) and talk to us.
```

Comment:
Fran created also milestone in the `spec` repo for `3.0.0` release -> https://github.com/asyncapi/spec/milestone/18 You should also point to it here.

Reply:
good point, added